frontend/src/operator/intercept/js/server_connector.js

```diff
@@ -227,22 +235,39 @@ class PollingConnector extends Connector {
       return;
     }
 
-    let currentPollDelay = 1000;
+    const shortPollDelay = 1000;
+
+    let currentPollDelay = shortPollDelay;
     let pollerRoutine = async () => {
       let messages = await this.#pollMessages();
 
-      // Do exponential backoff on the polling up to 60 seconds
-      currentPollDelay = Math.min(60000, 2 * currentPollDelay);
+      if (Date.now() < this.#pollQuicklyUntil) {
+        currentPollDelay = shortPollDelay;
+      } else {
+        // Do exponential backoff on the polling up to 60 seconds
+        currentPollDelay = Math.min(60000, 2 * currentPollDelay);
+      }
```

Comment:
nit: Extract this to a helper function named like `calcNextPollDelay()`.

Reply:
Done. PTAL!